src/header.tsx

```diff
@@ -175,81 +187,95 @@ export const HeaderLayout: React.FC<HeaderLayoutProps> = ({
+        <div>
+            {breadcrumbs && <div className={styles.breadcrumbs}>{breadcrumbs}</div>}
+            {header}
+        </div>
+    );
 
     return (
-        <>
-            <ResponsiveLayout
-                isInverse={isInverse}
-                dataAttributes={{'component-name': 'HeaderLayout', ...dataAttributes}}
-            >
+        <div {...getPrefixedDataAttributes({'component-name': 'HeaderLayout', ...dataAttributes})}>
+            <ResponsiveLayout isInverse={isInverse}>
                 <OverscrollColor />
-                {isTabletOrSmaller ? (
-                    <Box paddingTop={header ? 32 : 0} paddingBottom={24}>
-                        <Stack space={24}>
-                            {header}
-                            {!isBleedActivated && extra}
-                        </Stack>
-                    </Box>
-                ) : sideBySideExtraOnDesktop ? (
-                    <Box paddingTop={breadcrumbs ? 16 : 48} paddingBottom={48}>
+                <Box
+                    paddingTop={
+                        noPaddingY
+                            ? 0
+                            : {
+                                  mobile: header ? 32 : 0,
+                                  desktop: breadcrumbs ? 16 : 48,
+                              }
+                    }
+                    paddingBottom={
+                        noPaddingY
+                            ? 0
+                            : {
+                                  mobile: 24,
+                                  desktop: isBleedActivated && !sideBySideExtraOnDesktop ? 32 : 48,
+                              }
+                    }
+                >
+                    {sideBySideExtraOnDesktop ? (
                         <GridLayout
                             template="6+6"
-                            left={
-                                <Stack space={32}>
-                                    {breadcrumbs}
-                                    {header}
-                                </Stack>
+                            left={mainContent}
+                            right={
+                                <div className={isBleedActivated ? styles.hideOnTabletOrSmaller : ''}>
+                                    <Box paddingTop={{mobile: header ? 24 : 0, desktop: 0}}>{extra}</Box>
+                                </div>
                             }
-                            right={extra}
                         />
-                    </Box>
-                ) : (
-                    <Box paddingTop={breadcrumbs ? 16 : 48} paddingBottom={isBleedActivated ? 32 : 48}>
-                        <Stack space={isTabletOrSmaller ? 24 : 32}>
-                            <Stack space={32}>
-                                {breadcrumbs}
-                                {header}
-                            </Stack>
+                    ) : (
+                        <Stack space={header ? {mobile: 24, desktop: 32} : 0}>
```

Comment:
When Bleed and noPaddingY property are enabled, the padding between extra and description disappear, I think this is not expected

https://github.com/Telefonica/mistica-web/assets/6722153/e2400862-71f1-440e-8869-c5d9f59bbaa5

Reply:
fixed, good catch!